### DISCUSSION TITLE
Print CSS: avoid page break in some relevant cases

### DISCUSSION THREAD

**Discussion by rjoly** [pradyunsg/furo#520] [Ideas] [answered]
While printing a Sphinx documentation page as PDF, I noticed that a simple list (`<ul>`) was separated on two successive pages. 

I think simple CSS print rules could improve that kind of issues (avoid break inside a list, but also inside or after a hX title, etc.). 

You can see PR #519 for an implementation proposal.

**Comment by hmaarrfk** [comment]
Thank you for helping on this point!!!

I noticed that today it seems that headers and even text can be cut midway through. 

![image](https://github.com/user-attachments/assets/ce9f7e2e-6395-4e9e-8537-1b0be1579751)

To recreate:

1. Firefox
2. Linux (Ubuntu 24.04)
3. Go to https://pradyunsg.me/furo/kitchen-sink/generic/
4. Print
You can see in the screenshot that the text `Some menu ‣ Some sub menu 1 ‣ Some sub menu 2 ‣ Some sub menu 3` is literally cut between two pages.

I'm no CSS expert (or even a novice) so I can't contribute with code, but I will say that the ability to print is somewhat nice in science when you can always use an other screen to learn about something new.

**Comment by pradyunsg** [comment, marked as answer]
Along with #519, I've also added https://github.com/pradyunsg/furo/commit/fd3f422a0f30b3a2a9a57118532aabed59596d1e in response to the feedback that @hmaarrfk just shared.

**Reply by hmaarrfk** [reply]
Thanks for the quick fix. does it make sense to add `tr` to that same line? I'm trying your main branch locally and i noticed that I had a table row that was cut off in the middle in the same way. 

I tried to make the modification but it didn't seem to work maybe there is some caching issue locally for my testing.
